compiler/src/dotty/tools/dotc/tastyreflect/TreeOpsImpl.scala

```diff
@@ -12,6 +12,7 @@ trait TreeOpsImpl extends scala.tasty.reflect.TreeOps with CoreImpl with Helpers
   def TreeDeco(tree: Tree): TreeAPI = new TreeAPI {
     def pos(implicit ctx: Context): Position = tree.sourcePos
     def symbol(implicit ctx: Context): Symbol = tree.symbol
+    def withPos(pos: Position)(implicit ctx: Context): Unit = tree.withSpan(pos.span)
```

Comment:
Try rebasing on top of #5940. This should fix the missing positions assertions.

Reply:
Thanks a lot. BTW, do we want to keep this method?

Reply:
I would remove it for now. We can add it later.